webdriverio/webdriver/examples/deque/test/specs/swag.items.accessibility.spec.ts

```diff
@@ -0,0 +1,16 @@
+import {setTestContext} from "../helpers";
+import {LOGIN_USERS, PAGES} from "../configs/e2eConstants";
+
+describe('Swag Items Accessibility Spec', () => {
+
+    it('should accessibility scan the swag items page', async () => {
+        //Navigate to the page that you'd like to scan
+        await setTestContext({
+            user: LOGIN_USERS.STANDARD,
+            path: PAGES.SWAG_ITEMS
+        });
+        //Scans the page for accessibility and pushes results to sauce labs
+        await browser.getAxeResults();
+    });
+
```

Comment:
nitpicking, but maybe remove this line break

Reply:
removed line break